Libraries/Components/WebView/WebView.android.js

```diff
@@ -268,6 +272,14 @@ class WebView extends React.Component {
     );
   };
 
+  postMessage = (message) => {
+    UIManager.dispatchViewManagerCommand(
+      this.getWebViewHandle(),
+      UIManager.RCTWebView.Commands.postMessage,
+      [JSON.stringify(message)]
```

Comment:
On iOS, we just send the message as an object, and do the JSON stuff in Objective-C. I'd like the approaches on Android and iOS to be the same, so either we need to do the JSON stuff in Java (I couldn't figure out how), do the JSON stuff in JavaScript on iOS, or just enforce the message being a string and let the user decide how to transfer the data.